code/Forms/UnsavedChangesIndicator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace SilverStripe\Admin\Forms;
+
+use RuntimeException;
+use SilverStripe\Forms\FormAction;
+
+/**
+ * A visual indicator to show whether a form has unsaved changes.
+ *
+ * Note that this isn't strictly a FormAction as it doesn't perform any action when clicked.
+ * However, extending FormAction allows us to easily integrate with the existing form action
+ * architecture (e.g. placement in the form actions area).
+ */
+class UnsavedChangesIndicator extends FormAction
+{
+    private static array $minutes = [
```

Comment:
```suggestion
    /**
     * An associative array of time periods to determine when the notice displays, and when it updates to a warning.
     * notice: The number of minutes it takes to display the notice after starting to edit a form.
     * warning: The number of minutes it takes to update the display to a warning after displaying the notice.
     */
    private static array $minutes = [
```
Sorry, should have asked for this the first time around.

Reply:
Updated